### ISSUE TITLE
In Docs / Forms / Validation, No programmatic link between error messages dans input fields

### ISSUE THREAD

**Issue by Aniort** [twbs/bootstrap#37405]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

Hello,
first, there's no visual indication of required fields (like a "*" or something else but explained in text before the form ex.: "* Required f'ields"),
it's a violation of WCAG 3.3.2 Labels or instructions,
There is no programmatic link between error messages and input fields after a form validation,
it's a violation of WCAG 3.3.1 Error Identification,
to correct this , you can add an aria-labelledby with the id of the label and the id of the error message,
may also use an aria-invalid="true"

### Reduced test cases

https://getbootstrap.com/docs/5.2/forms/validation/

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Chrome, Firefox

### What version of Bootstrap are you using?

5.2.x

**Comment by patrickhlauke** [comment]
This is in progress here https://github.com/twbs/bootstrap/pull/34043

**Comment by patrickhlauke** [comment]
duplicate of https://github.com/twbs/bootstrap/issues/28414
